src/mapd-crossfilter.js

```diff
@@ -1318,16 +1309,25 @@ export function replaceRelative(sqlStr) {
             subExpression += " AND "
           }
 
-          var typedRange =
-            range[e][0] instanceof Date
-              ? [
-                  formatDateRangeLowerBound(range[e][0]),
-                  formatDateRangeUpperBound(range[e][1])
-                ]
-              : [
-                  formatFilterValue(range[e][0], true),
-                  formatFilterValue(range[e][1], true)
-                ]
+          const dateRange = range[e][0] instanceof Date
```

Comment:
If this is a boolean indicating whether a given range is a date, can we name it something more like `rangeIsDate`?

Reply:
Good call, definitely clearer.